engine/samples/cars/components.hpp

```diff
@@ -2,8 +2,10 @@
 
 #include <glm/glm.hpp>
 
+/// [Car component]
 struct [[cubos::component("car", VecStorage)]] Car
```

Comment:
```suggestion
struct [[cubos::component("car")]] Car
```

The storage isn't needed anymore, defaults to `VecStorage` :relieved: